docs/source/tutorials/staging_data.ipynb

```diff
@@ -1,20 +1,419 @@
 {
```

Comment:
<p>It is known that the I/O operations are expensive, epecially in terms of spent time. -&gt; It could be good to either cite an article, or website, or give a concrete example. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>